### DISCUSSION TITLE
QueuedRunCoordinator configuration

### DISCUSSION THREAD

**Discussion by yuval-airis** [dagster-io/dagster#17491] [Q&A] [answered]
Looking [at dagster.yaml snippets](https://github.com/dagster-io/dagster/blob/8e12a1f5ee5b117c43c8e5ab33234e32c4756d0f/examples/docs_snippets/docs_snippets/deploying/dagster_instance/dagster.yaml#L165) I see that QueuedRunCoordinator can have max_concurrent_runs and dequeue_num_workers. What is the difference between the two? is the deque and the handling of dequed elements separate? When would I not want these two to be the same?

**Comment by sryza** [comment, marked as answer]
Hey @yuval-airis - the API doc includes details on each of the configuration parameters: https://docs.dagster.io/_apidocs/internals#dagster._core.run_coordinator.QueuedRunCoordinator

`dequeue_num_workers` refers to the number of worker threads inside the run coordinator. These are threads that are just responsible for dequeuing runs and launching them. They don't carry out the execution of the runs themselves.
